test/second_q/circuit/library/ansatzes/test_succd.py

```diff
@@ -150,3 +150,131 @@ def test_puccd_ansatz_generalized(self, num_spin_orbitals, num_particles, expect
+            ],
+        ),
+        (
+            6,
+            (1, 1),
+            (True, False),
+            [
+                FermionicOp([("+-IIII", 1j), ("-+IIII", 1j)], display_format="dense"),
+                FermionicOp([("+I-III", 1j), ("-I+III", 1j)], display_format="dense"),
+                FermionicOp([("+-I+-I", 1j), ("-+I-+I", (-0 - 1j))], display_format="dense"),
+                FermionicOp(
+                    [("+-I+I-", 1j), ("-+I-I+", (-0 - 1j)), ("+I-+-I", 1j), ("-I+-+I", (-0 - 1j))],
+                    display_format="dense",
+                ),
+                FermionicOp([("+I-+I-", 1j), ("-I+-I+", (-0 - 1j))], display_format="dense"),
+            ],
+        ),
+        (
+            6,
+            (1, 1),
+            (False, True),
+            [
+                FermionicOp([("III+-I", 1j), ("III-+I", 1j)], display_format="dense"),
+                FermionicOp([("III+I-", 1j), ("III-I+", 1j)], display_format="dense"),
+                FermionicOp([("+-I+-I", 1j), ("-+I-+I", (-0 - 1j))], display_format="dense"),
+                FermionicOp(
+                    [("+-I+I-", 1j), ("-+I-I+", (-0 - 1j)), ("+I-+-I", 1j), ("-I+-+I", (-0 - 1j))],
+                    display_format="dense",
+                ),
+                FermionicOp([("+I-+I-", 1j), ("-I+-I+", (-0 - 1j))], display_format="dense"),
+            ],
+        ),
+    )
+    def test_succ_full_with_singles(
+        self, num_spin_orbitals, num_particles, include_singles, expect
+    ):
+        """Tests the SUCCD_full Ansatz with included single excitations."""
+        converter = QubitConverter(JordanWignerMapper())
+
+        ansatz = SUCCD(
+            qubit_converter=converter,
+            num_particles=num_particles,
+            num_spin_orbitals=num_spin_orbitals,
+            include_singles=include_singles,
+            mirror=True,
+        )
+
+        assert_ucc_like_ansatz(self, ansatz, num_spin_orbitals, expect)
+
+    @unpack
+    @data(
+        (
+            6,
+            (1, 1),
+            [
+                FermionicOp([("+-I+-I", 1j), ("-+I-+I", (-0 - 1j))], display_format="dense"),
+                FermionicOp(
+                    [("+-I+I-", 1j), ("-+I-I+", (-0 - 1j)), ("+I-+-I", 1j), ("-I+-+I", (-0 - 1j))],
+                    display_format="dense",
+                ),
+                FermionicOp([("+-II+-", 1j), ("-+II-+", (-0 - 1j))], display_format="dense"),
+                FermionicOp([("+I-+I-", 1j), ("-I+-I+", (-0 - 1j))], display_format="dense"),
+                FermionicOp([("+I-I+-", 1j), ("-I+I-+", (-0 - 1j))], display_format="dense"),
+                FermionicOp([("I+-+-I", 1j), ("I-+-+I", (-0 - 1j))], display_format="dense"),
+                FermionicOp([("I+-+I-", 1j), ("I-+-I+", (-0 - 1j))], display_format="dense"),
+                FermionicOp([("I+-I+-", 1j), ("I-+I-+", (-0 - 1j))], display_format="dense"),
```

Comment:
This is actually not yet the desired behavior. The expected outcome should be this:
```suggestion
                FermionicOp([("+-I+-I", 1j), ("-+I-+I", (-0 - 1j))], display_format="dense"),
                FermionicOp(
                    [("+-I+I-", 1j), ("-+I-I+", (-0 - 1j)), ("+I-+-I", 1j), ("-I+-+I", (-0 - 1j))],
                    display_format="dense",
                ),
                FermionicOp(
                    [("+-II+-", 1j), ("-+II-+", (-0 - 1j)), ("I+-+-I", 1j), ("I-+-+I", (-0 - 1j))],
                    display_format="dense",
                ),
                FermionicOp([("+I-+I-", 1j), ("-I+-I+", (-0 - 1j))], display_format="dense"),
                FermionicOp(
                    [("+I-I+-", 1j), ("-I+I-+", (-0 - 1j)), ("I+-+I-", 1j), ("I-+-I+", (-0 - 1j))],
                    display_format="dense",
                ),
                FermionicOp([("I+-I+-", 1j), ("I-+I-+", (-0 - 1j))], display_format="dense"),
```